Podfile.lock

```diff
@@ -59,4 +59,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 6ee09db13e81dfe3abdf89ad811b4de542da1a35
 
-COCOAPODS: 1.2.1
+COCOAPODS: 1.3.1
```

Comment:
Are we all on this version?

Reply:
Huh. I did not notice that being updated. Can revert, if it's supposed to be `1.2.1`.

Reply:
Not a problem. I just hate it when every PR changes the pods version ;)